docs/docs/writing-hello-world.md

```diff
@@ -464,42 +406,30 @@ public class NativeTest {
 }
 ```
 
-You'll notice that we annotated the test class with `@EnabledOnOs(OS.LINUX)`. This is from
-[JUnit 5](https://junit.org/junit5/docs/current/user-guide/#writing-tests-conditional-execution-os) and it will make sure
-the native test isn't run on non-Linux environments.
-
-The tests can use any enclave mode: release, debug, simulation or mock. Therefore, if you want to run your tests on
-a non-Linux system then you can configure your tests to depend on a mock mode enclave instead. In this case, remove the
-`@EnabledOnOs(OS.LINUX)` annotation from the above code.
+The test class is annotated with `@EnabledOnOs(OS.LINUX)`. This is from
+[JUnit 5](https://junit.org/junit5/docs/current/user-guide/#writing-tests-conditional-execution-os) to ensure that the
+native test is run only on a Linux environment.
 
-!!! note
-    Running your integration tests in mock mode is very similar to
-    [Integrating enclave tests in your host project](#integrating-enclave-tests-in-your-host-project). In both cases
-    the enclave code is loaded and run fully in memory. However, for integration tests, you can also choose to run
-    the tests in modes other than mock, which is not possible for enclave project tests.
+To execute the test using a simulation enclave, run:
 
-Running
-
-```
+```bash
 ./gradlew host:test
 ```
 
-will execute the test using a simulation enclave, or not at all if the OS is not Linux. You can switch to a debug enclave
-and test on real secure hardware by using the `-PenclaveMode` flag:
+To switch to a debug enclave and test on real secure hardware, use the `-PenclaveMode` flag:
 
-```
+```bash
 ./gradlew -PenclaveMode=debug host:test
 ```
 
-Or you can use a mock enclave and test on a non-Linux platform by removing `@EnabledOnOs(OS.LINUX)` and by running this
-command:
+To use a mock enclave and test on a non-Linux platform, remove `@EnabledOnOs(OS.LINUX)` and run:
 
-```
+```bash
 ./gradlew -PenclaveMode=mock host:test
 ```
 
```

Comment:
Likewise for the native tests: https://github.com/R3Conclave/conclave-tutorials/blob/HEAD/hello-world/host/src/test/java/com/r3/conclave/sample/host/NativeTest.java